pandas/tests/series/apply/test_series_apply.py

```diff
@@ -222,7 +223,7 @@ def test_transform(self, string_series):
             expected.columns = ["sqrt"]
             tm.assert_frame_equal(result, expected)
 
-            result = string_series.transform([np.sqrt])
+            result = string_series.apply([np.sqrt])
```

Comment:
The line `transform([np.sqrt])` occurs 5 lines up. I believe the original intention was to test `apply` here.